frontend/tailwind.config.ts

```diff
@@ -9,7 +9,6 @@ export default {
 		"./node_modules/flowbite-svelte/**/*.{html,js,svelte,ts}",
 	],
 	darkMode: "selector",
-	// ensure svelte compiler doesn't optimize these away if it doesn't realise we are using them
```

Comment:
could we keep this comment? (otherwise this safelist looks pointless and I would probably delete it at some point in the future :-))